roles/gpu_operator_deploy_from_operatorhub/tasks/deploy_from_catalog.yml

```diff
@@ -26,13 +26,23 @@
       > {{ artifact_extra_logs_dir }}/gpu_operator_packagemanifest.json
 
   rescue:
+  - name: Capture the Catalog Operator logs (debug)
```

Comment:
Is this related to this PR?

Reply:
not related to the title of this PR, but separated into a dedicated commit (https://github.com/openshift-psap/ci-artifacts/pull/192/commits/fbad26a7b1f6c60324204f77c37449e84c2d8ed2) [and I had to update the commits because if got messed up]

I usually ship minor fixes along with the bigger PRs to avoid extra rounds of OCP testing (that's fully incompatible is squash-merging, but the default here is rebase-merging)